Makefile

```diff
@@ -48,7 +48,7 @@ test:
 	go test -race ./...
 
 test-e2e:
-	go test -race -mod=readonly --failfast -timeout=25m -v $(PACKAGES_E2E) -count=1 --tags=e2e
+	go test -mod=readonly --failfast -timeout=25m -v $(PACKAGES_E2E) -count=1 --parallel 10 --tags=e2e
```

Comment:
Removed the `-race` flag, as there were quite a bit of race conditions not related to the testing code, but some in Babylon some in cosmos SDK. Will open an issue to investigate this

-parallel 10 -> limits the number of concurrent tests to 10, we can play around with this, will depend on how much we can stress GH actions, or remove the limit